### ISSUE TITLE
Flutter regex match also pub packages

### ISSUE THREAD

**Issue by GreenDjango** [aziznal/dart-import-sorter#33]
Hi, you can add a slash after the `flutter` word like this:
```json
{
  "label": "Flutter",
  "regex": "^package:flutter/.*$",
  "order": 10,
  "regexFlags": ["m"]
}
```
It will prevent match package line `package:flutter_riverpod`

Thanks for this extension btw, much better than default import sort :+1:.

**Comment by aziznal** [comment]
Good catch. I'm very glad to hear the extension is useful for you.

**Comment by aziznal** [comment]
Requested changes have been implemented in latest version (0.2.6)
